### DISCUSSION TITLE
无法联网

### DISCUSSION THREAD

**Discussion by System32-edge** [OdysseusYuan/Mocreak#22] [Q&A] [answered]
我在安装Office时提示未连接到互联网，之前确实断开过,但是安装时已经连上了，现在无法安装，已经 _**卸载了好几遍**_
**求解答**

**Comment by OdysseusYuan** [comment]
感谢您的使用。Office 原版安装必须保持联网。
（1）如果您在虚拟机或 Docker 中安装，需要确保虚拟机网卡与宿主网卡桥接正常。
（2）如果安装环境为真机环境，且开启了网络代理，可暂时关闭，或务必确保代理环境正常。
（3）如果以上均正常，请尝试 ping baidu.com，如果无法 ping 通，可能您的网络不太稳定，可稍后再次尝试。

**Comment by OdysseusYuan** [comment, marked as answer]
您好，目前 Mocreak 已经发布了最新版本，进一步提升了网络环境检查的科学性，您可尝试使用。

下载地址：https://github.com/OdysseusYuan/Mocreak/releases/latest

以上，如有问题，欢迎随时沟通。
